grpc/client.go

```diff
@@ -76,5 +77,8 @@ func prepareClientContext(ctx context.Context) context.Context {
 	if reqID := log.RequestIDFromContext(ctx); reqID != "" {
 		ctx = metadata.AppendToOutgoingContext(ctx, "req_id", reqID)
 	}
+	if utmCtx, ok := utm.FromContext(ctx); ok {
+		ctx = metadata.AppendToOutgoingContext(ctx, "utm", utmCtx.ToString())
```

Comment:
Why don't you attach the values individually? How are they reconstructed on the server side?